src/HydraClient.ts

```diff
@@ -0,0 +1,124 @@
+import {hydra} from "./namespaces";
+import {IMetadataProvider} from "./DataModel/IMetadataProvider";
+import {IApiDocumentation} from "./DataModel/IApiDocumentation";
+import {IMetadataDescription} from "./DataModel/IMetadataDescription";
+import ApiDocumentation from "./ApiDocumentation";
+
+/**
+ * @class HydraClient Represents a Hypermedia Driven Application (HyDrA) Core Vocabulary client.
```

Comment:
Well, it could be something along the lines of Heracles is a generic client for Hydra-powered Web APIs. To learn more about Hydra ..."